src/main/scala/cromwell/binding/values/WdlBoolean.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell.binding.values
+
+import cromwell.binding.WdlExpressionException
+import cromwell.binding.types.WdlBooleanType
+
+case class WdlBoolean(value: Boolean) extends WdlPrimitive {
+  val wdlType = WdlBooleanType
+  override def equals(rhs: WdlValue): WdlBoolean = {
+    rhs match {
+      case r:WdlBoolean => WdlBoolean(value == r.value)
+      case _ => invalid(s"$value || $rhs")
+    }
+  }
+  override def lessThanOrEqual(rhs: WdlValue): WdlBoolean = equals(rhs)
+  override def greaterThanOrEqual(rhs: WdlValue): WdlBoolean = equals(rhs)
```

Comment:
My vote is for `Int + Boolean` to be an error but `Boolean > Boolean` to work.  The comparison operators on booleans are pretty consistent across languages that I've checked so far... so if somebody happens to do it in WDL (for whatever reason) it'd follow the principle of least surprise I think.  Plus it's so edge-casey that I doubt it'll really matter either way (considering that none of us even knew how Scala handled it!)